config/constants.ts

```diff
@@ -0,0 +1,48 @@
+import { OrcaBusStatefulConfig } from '../lib/workload/orcabus-stateful-stack';
+import { AuroraPostgresEngineVersion } from 'aws-cdk-lib/aws-rds';
+import { OrcaBusStatelessConfig } from '../lib/workload/orcabus-stateless-stack';
+
+const regName: string = 'OrcaBusSchemaRegistry';
+
+export const orcaBusStatefulConfig: OrcaBusStatefulConfig = {
+  schemaRegistryProps: {
+    registryName: regName,
+    description: 'Registry for OrcaBus Events',
+  },
+  eventBusProps: {
+    eventBusName: 'OrcaBusMain',
+    archiveName: 'OrcaBusMainArchive',
+    archiveDescription: 'OrcaBus main event bus archive',
+    archiveRetention: 365,
+  },
+  databaseProps: {
+    clusterIdentifier: 'orcabus-db',
+    defaultDatabaseName: 'orcabus',
```

Comment:
Same as for the bus. I suppose the DB name will be needed as well in the stateless stack?

Reply:
Good point! It might be more like db SM that need externalise.. I will investigate